2_base_and_sensor.ipynb

```diff
@@ -14,19 +16,56 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# Controlling the move base and captureing sensor information\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Objective</b>\n",
+    "    <p>In this notebook, we will learn how to control the move base of HSR and how to access sensor information.</p>\n",
```

Comment:
"    <p>In this notebook, we will learn how to control the HSR's moving base and access sensor information.</p>\n",